src/App.jsx

```diff
@@ -1,22 +1,23 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
 import {Routes, Route} from 'react-router-dom'
-import Articles from './components/All_Articles'
+import Articles from './components/All_Articles/All_Articles'
 import Header from './components/Header'
 import Profile from './components/Profile'
-import ArticleID from './components/ArticleID'
+import IndividualArticle from './components/Individual_Article/Individual_Article'
+import {useState} from 'react'
+
 
 function App() {
   
+  const [articleId, setArticle] = useState(null)
```

Comment:
There is zero need for this state to be in app. You can get this information from the url.